test/commonControlTests/NumberControl.spec.ts

```diff
@@ -11,608 +11,436 @@
-                    }),
-                ),
-                'A: Was that 16?',
-                'U: Yes, change it to 6',
-                TestInput.of(
-                    ValueControlIntent.of(AmazonBuiltInSlotType.NUMBER, {
-                        feedback: $.Feedback.Affirm,
-                        action: $.Action.Change,
-                        'AMAZON.NUMBER': '6',
-                    }),
-                ),
-                'A: Sorry the previous value is 16, but you confirmed to a different number. Was that 6?',
-                'U: Yes.',
-                TestInput.of(IntentBuilder.of(AmazonIntent.YesIntent)),
-                'A: Great.',
-            ]);
-            expect(
-                (requestHandler.getSerializableControlStates()[TEST_CONTROL_ID] as NumberControlState).value,
-            ).eq(6);
-        });
+    test('change age value with explicit confirmation', async () => {
+        const requestHandler = new ControlHandler(new AgeControlManager());
+        await testE2E(requestHandler, [
+            'U: change my to four',
+            TestInput.of(
+                ValueControlIntent.of(AmazonBuiltInSlotType.NUMBER, {
+                    'AMAZON.NUMBER': '4',
+                    action: $.Action.Change,
+                }),
+            ),
+            'A: Was that 4?',
+            'U: yes',
+            TestInput.of(IntentBuilder.of(AmazonIntent.YesIntent)),
+            'A: Great.',
+        ]);
+        expect(requestHandler.getSerializableControlStates().ageSelector.value).eq(4);
+    });
 
-        test('disaffirm with the same value, confirm user with the number', async () => {
-            const requestHandler = new ControlHandler(new NumberControlManager());
-            await testE2E(requestHandler, [
-                'U: Sixteen',
-                TestInput.of(
-                    ValueControlIntent.of(AmazonBuiltInSlotType.NUMBER, {
-                        action: $.Action.Set,
-                        'AMAZON.NUMBER': '16',
-                    }),
-                ),
-                'A: Was that 16?',
-                'U: No, change it to 16',
-                TestInput.of(
-                    ValueControlIntent.of(AmazonBuiltInSlotType.NUMBER, {
-                        feedback: $.Feedback.Disaffirm,
-                        action: $.Action.Change,
-                        'AMAZON.NUMBER': '16',
-                    }),
-                ),
-                'A: Sorry you disagreed but with the same number. How many?',
-                'U: Six.',
-                TestInput.of(ValueControlIntent.of(AmazonBuiltInSlotType.NUMBER, { 'AMAZON.NUMBER': '6' })),
-                'A: Ok. Value set to 6.',
-            ]);
-            expect(
-                (requestHandler.getSerializableControlStates()[TEST_CONTROL_ID] as NumberControlState).value,
-            ).eq(6);
-        });
+    test('change age value with explicit disaffirmation', async () => {
```

Comment:
name suggestion `change age value with explicit disaffirmation *and no suggestion available*`